### ISSUE TITLE
Can't start a workspace on OpenShift after #18121

### ISSUE THREAD

**Issue by sparkoo** [eclipse-che/che#18157]
### Describe the bug

Can't start any workspace on openshift. It fails with
```
Error: Failed to run the workspace: "Strategy of generating ingress URLs for Che servers is set to 'multi-host', but property 'che.infra.kubernetes.ingress.domain' is not set"
```

I think that issue is caused by https://github.com/eclipse/che/pull/18121, which introduces `MultiHostExternalServiceExposureStrategy` into `JwtProxy` provisioners. The `MultiHostExternalServiceExposureStrategy` class requires `che.infra.kubernetes.ingress.domain` property https://github.com/eclipse/che/blob/master/infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/server/external/MultiHostExternalServiceExposureStrategy.java#L56 which is not set on OpenShift.

### Che version

 - [x] current master after https://github.com/eclipse/che/pull/18121

### Steps to reproduce

1. deploy latest master on OpenShift (or anything after https://github.com/eclipse/che/pull/18121)
1. try run any workspace

### Expected behavior

Workspace must start

### Runtime
   - [x] Openshift (include output of `oc version`)

### Installation method
   - [x] chectl

### Environment
   - [x] my computer
       - [x] Linux

### Eclipse Che Logs

```
2020-10-21 12:13:56,449[aceSharedPool-1]  [ERROR] [o.e.c.a.w.s.WorkspaceRuntimes 994]   - Strategy of generating ingress URLs for Che servers is set to 'multi-host', but property 'che.infra.kubernetes.ingress.domain' is not set
org.eclipse.che.api.workspace.server.spi.InternalInfrastructureException: Strategy of generating ingress URLs for Che servers is set to 'multi-host', but property 'che.infra.kubernetes.ingress.domain' is not set
	at org.eclipse.che.workspace.infrastructure.kubernetes.KubernetesInternalRuntime.wrapAndRethrow(KubernetesInternalRuntime.java:963)
	at org.eclipse.che.workspace.infrastructure.kubernetes.KubernetesInternalRuntime.internalStart(KubernetesInternalRuntime.java:275)
	at org.eclipse.che.api.workspace.server.spi.InternalRuntime.start(InternalRuntime.java:141)
	at org.eclipse.che.api.workspace.server.WorkspaceRuntimes$StartRuntimeTask.run(WorkspaceRuntimes.java:960)
	at org.eclipse.che.commons.lang.concurrent.CopyThreadLocalRunnable.run(CopyThreadLocalRunnable.java:38)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: org.eclipse.che.inject.ConfigurationException: Strategy of generating ingress URLs for Che servers is set to 'multi-host', but property 'che.infra.kubernetes.ingress.domain' is not set
	at org.eclipse.che.workspace.infrastructure.kubernetes.server.external.MultiHostExternalServiceExposureStrategy.<init>(MultiHostExternalServiceExposureStrategy.java:58)
	at org.eclipse.che.workspace.infrastructure.kubernetes.server.external.MultiHostExternalServiceExposureStrategy$$FastClassByGuice$$976df250.newInstance(<generated>)
	at com.google.inject.internal.DefaultConstructionProxyFactory$FastClassProxy.newInstance(DefaultConstructionProxyFactory.java:89)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:114)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:306)
	at com.google.inject.internal.SingleParameterInjector.inject(SingleParameterInjector.java:42)
	at com.google.inject.internal.SingleParameterInjector.getAll(SingleParameterInjector.java:65)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:113)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:306)
	at com.google.inject.internal.InjectorImpl$1.get(InjectorImpl.java:1050)
	at com.google.inject.assistedinject.FactoryProvider2.invoke(FactoryProvider2.java:836)
	at com.sun.proxy.$Proxy61.create(Unknown Source)
	at org.eclipse.che.workspace.infrastructure.kubernetes.server.secure.DefaultSecureServerExposer.<init>(DefaultSecureServerExposer.java:65)
	at org.eclipse.che.workspace.infrastructure.kubernetes.server.secure.jwtproxy.JwtProxySecureServerExposer.<init>(JwtProxySecureServerExposer.java:52)
	at org.eclipse.che.workspace.infrastructure.kubernetes.server.secure.jwtproxy.JwtProxySecureServerExposer$$FastClassByGuice$$e0111ce7.newInstance(<generated>)
	at com.google.inject.internal.DefaultConstructionProxyFactory$FastClassProxy.newInstance(DefaultConstructionProxyFactory.java:89)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:114)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:306)
	at com.google.inject.internal.InjectorImpl$1.get(InjectorImpl.java:1050)
	at com.google.inject.assistedinject.FactoryProvider2.invoke(FactoryProvider2.java:836)
	at com.sun.proxy.$Proxy62.create(Unknown Source)
	at org.eclipse.che.workspace.infrastructure.kubernetes.provision.server.ServersConverter.provision(ServersConverter.java:66)
	at org.eclipse.che.workspace.infrastructure.openshift.OpenShiftEnvironmentProvisioner.provision(OpenShiftEnvironmentProvisioner.java:149)
	at org.eclipse.che.workspace.infrastructure.openshift.OpenShiftEnvironmentProvisioner.provision(OpenShiftEnvironmentProvisioner.java:57)
	at org.eclipse.che.workspace.infrastructure.kubernetes.wsplugins.PluginBrokerManager.getTooling(PluginBrokerManager.java:118)
	at org.eclipse.che.workspace.infrastructure.kubernetes.wsplugins.SidecarToolingProvisioner.provision(SidecarToolingProvisioner.java:90)
	at org.eclipse.che.workspace.infrastructure.kubernetes.KubernetesInternalRuntime.provisionWorkspace(KubernetesInternalRuntime.java:286)
	at org.eclipse.che.workspace.infrastructure.openshift.OpenShiftInternalRuntime.provisionWorkspace(OpenShiftInternalRuntime.java:131)
	at org.eclipse.che.workspace.infrastructure.kubernetes.KubernetesInternalRuntime.internalStart(KubernetesInternalRuntime.java:202)
	... 7 common frames omitted
```

### Additional context

https://github.com/eclipse/che/pull/18121

**Comment by metlos** [comment]
For now, fixed by https://github.com/eclipse/che/pull/18161.

A new pull request that fixes #18065 and works on OpenShift will come shortly(-ish).
